vertical-pod-autoscaler/pkg/utils/vpablessing/vpa_blessing_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vpablessing
+
+import (
+	"fmt"
+	"testing"
+
+	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/utils/test"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+)
+
+func TestGetVpaBlessingAnnotationValue(t *testing.T) {
+	tests := []struct {
+		name string
+		pod  *v1.Pod
+		want string
+	}{
+		{
+			name: "creating blessing annotation",
+			pod: test.Pod().
+				AddContainer(test.Container().WithName("test1").Get()).
+				AddContainer(test.Container().WithName("test2").Get()).
+				AddContainer(test.Container().WithName("test3").Get()).
+				Get(),
+			want: "[test1, test2, test3]",
+		},
+	}
+	for _, tc := range tests {
+		t.Run(fmt.Sprintf("test case: %s", tc.name), func(t *testing.T) {
+			got := GetVpaBlessingAnnotationValue(tc.pod)
+			assert.Equal(t, got, tc.want, fmt.Sprintf("got %s, want %s", got, tc.want))
```

Comment:
nit: The string you want to print on test failure seems redundant, isn't this the exact output of Equal in case of inequality?

Reply:
Right. Simplified.